etc/kayobe/environments/ci-multinode/inventory/group_vars/all/stackhpc-lvm.yml

```diff
@@ -33,16 +33,16 @@ stackhpc_lvm_group_rootvg_lvs_default:
 
```

Comment:
This file should not be modified. It should be the same as etc/kayobe/inventory/group_vars/all/stackhpc-lvm in stackhpc/xena